kasa/feature.py

```diff
@@ -172,7 +172,15 @@ async def set_value(self, value):
         return await getattr(container, self.attribute_setter)(value)
 
     def __repr__(self):
-        value = self.value
+        if self.type == Feature.Type.Choice:
+            value = self.choices.copy()
+            val = self.value
+            index = value.index(val)
+            value.remove(val)
+            value.insert(index, f"*{val}*")
+            value = " ".join(value)
```

Comment:
I'd rather do this using a list comprehension to avoid mutating the list, something like:
```
value = " ".join([f"*{choice}* if choice == value else choice for choice in self.choices])
```

Reply:
Ai, that's so much better, done.